### DISCUSSION TITLE
Values inside datepicker range don't display

### DISCUSSION THREAD

**Discussion by vitali-duckma** [ant-design/ant-design#40330] [Q&A]
https://codesandbox.io/s/select-range-max-today-tpsjy0

Hi guys, I followed the antd docs, but wanted to create and object of **from** and **to** date. Everything works fine excepting the values are not displayed, anyone can help please? Thanks.

**Comment by zombieJ** [comment]
value should be dayjs object. You provide `string` instead which is not dayjs obj.

**Reply by vitali-duckma** [reply]
So what I should pass to value ?? 
`{dayjs.(dates.from).toDate()}` ??
